demo/forms.json

```diff
@@ -1,9 +1,4 @@
 [
-  {
-    "title": "Example Form",
-    "src": "./assets/example/form.html",
-    "id": "example"
-  },
   {
     "title": "Class Registration",
```

Comment:
Because this forms list is specific to the Class demo, perhaps we namespace it to `class-forms.json`? Not a big deal right now but we should keep thinking about how we organize demos.